packages/turf-interpolate/index.js

```diff
@@ -0,0 +1,64 @@
+var meta = require('@turf/meta');
+var bbox = require('@turf/bbox');
+var poinGrid = require('@turf/point-grid');
+var distance = require('@turf/distance');
+var invariant = require('@turf/invariant');
+var featureEach = meta.featureEach;
+var collectionOf = invariant.collectionOf;
+
+/**
+ * Takes a set of points and estimates their 'property' values on a grid using the Inverse Distance Weighting (IDW) method.](https://en.wikipedia.org/wiki/Inverse_distance_weighting.
+ *
+ * @name interpolate
+ * @param {FeatureCollection<Point>} points with known value
+ * @param {number} cellSize the distance across each grid point
+ * @param {string} [property='elevation'] the property name in `points` from which z-values will be pulled
+ * @param {string} [units=kilometers] used in calculating cellSize, can be degrees, radians, miles, or kilometers
+ * @param {number} [weight=1] exponent regulating the distance-decay weighting
+ * @returns {FeatureCollection<Point>} grid of points with 'property'
+ * @example
+ * var points = turf.random('points', 30, {
+ *     bbox: [50, 30, 70, 50]
+ * });
+ * // add a random property to each point
+ * turf.featureEach(points, function(point) {
+ *     point.properties.solRad = Math.random() * 50;
+ * });
+ * var grid = interpolate(points, 100, 'solRad', 'miles');
+ *
+ * //addToMap
+ * var addToMap = grid
+ */
+module.exports = function (points, cellSize, property, units, weight) {
+    // validation
+    if (!points) throw new Error('points is required');
+    collectionOf(points, 'Point', 'input must contain Points');
+    if (!cellSize) throw new Error('cellSize is required');
+    if (cellSize === undefined || cellSize === null) throw new Error('cellWidth is required');
```

Comment:
I'm going to assume this is a duplicate of `cellSize`.

`CellSize` cannot be 0, correct?